analyzer/config.go

```diff
@@ -0,0 +1,33 @@
+package analyzer
+
+import (
+	"os"
+
+	"github.com/brimdata/zed/zio"
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	Args       []string       `yaml:"args"`
+	Cmd        string         `yaml:"cmd"`
+	Globs      []string       `yaml:"globs"`
+	Launcher   Launcher       `yaml:"-"`
+	ReaderOpts zio.ReaderOpts `yaml:"-"`
+	Shaper     string         `yaml:"shaper"`
+	StdoutPath string         `yaml:"stdout"`
+	StderrPath string         `yaml:"stderr"`
```

Comment:
```suggestion
	StdoutPath string         `yaml:"stdout_path"`
	StderrPath string         `yaml:"stderr_path"`
```

Reply:
Idk, don't you think stdout without path is easier to understand than `stdout_path`?